docs/admin/installing.rst

```diff
@@ -2,7 +2,78 @@
 Installing
 **********
 
+Installing required dependencies
+================================
+
+We use Python 3.6, so in order to use this software please install it
+into your environment beforehand. Note that an additional step is
+needed for Ubuntu releases older than 16.10.
+
+
+Python 3.6 in old Ubuntu releases
+=================================
+
+If are you using Ubuntu 16.04 or older, you must add a PPA to be able to
+install Python 3.6 packages. To add this PPA, use the commands:
+
+.. code-block:: bash
+
+  $ sudo add-apt-repository ppa:jonathonf/python-3.6
+  $ sudo apt update
+
+Required packages
+=================
+
+The required Ubuntu packages can be installed by:
+
+.. code-block:: bash
+
+  $ sudo apt install python-pip python3.6
+
+Installing with pip
+===================
+
+To install the latest Kytos from PyPI, make sure that you done all
+previous steps, then:
+
+.. code-block:: bash
+
+  $ pip install kytos
+
+This will install Kytos with all the dependencies, like kytos-utils and
+python-openflow.
 
 
 Production Environment
 ======================
+
+To download now the latest release (it still a beta software) from our
+repository, first you need to clone *kytos* repository. So, install git
+before clone:
```

Comment:
1. Instructions on how to install with Git should be moved to the dev guide, production environment should install with Pip.
2. Maybe we don't need to do this right now, but in the future we should have instructions on how to install kytos with a production web server.